src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+$second-text-color: #616070;
+$button-color: #00acdc;
+
+.card {
+  margin: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    margin: 32px 19px 40px;
```

Comment:
instead of margins to image would be better add paddings to `card `above and add `margin-bottom` to this image 40px